common/include/pcl/common/impl/common.hpp

```diff
@@ -70,6 +70,20 @@ pcl::getAngle3D (const Eigen::Vector3f &v1, const Eigen::Vector3f &v2, const boo
 inline void
 pcl::getMeanStd (const std::vector<float> &values, double &mean, double &stddev)
 {
+  // throw an exception when the input array is empty
+  if(values.size() == 0 )
+  {
+    PCL_THROW_EXCEPTION(BadArgumentException, "Error : Input array must have at least 1 element."); 
+  }
+  
+  // when the array has only one element, mean is the number itself and standard dev is 0
+  if(values.size() == 1)
+  {
+    mean = static_cast<double>(values.at(0));
```

Comment:
There is no reason for an explicit cast here, only makes the line harder to read.

Reply:
@taketwo did the asked changes. Please have a look.